allure-report-face/src/components/steps/styles.css

```diff
@@ -29,6 +29,12 @@
   &__time {
     color: $text-muted-color
   }
+  &__duration {
+    color: $text-muted-color;
+    position: absolute;
```

Comment:
I don't like position absolute at this point

<img width="433" alt="screen shot 2017-02-07 at 15 07 27" src="https://cloud.githubusercontent.com/assets/2149631/22690517/38747aca-ed47-11e6-89a4-b1f9241640a5.png">